Cannot manage to reproduce to use `with-playwright`

Hello everybody !

I'm facing the following issue:

**Describe the bug**
I'm not able to reproduce the [getting started instructions for Playwright](https://github.com/Zenika/alpine-chrome#how-to-use-with-playwright)

**To Reproduce**
Steps to reproduce the behavior:

1. Version of the image :
```
$ docker image inspect --format='{{index .RepoDigests 0}}' zenika/alpine-chrome:with-playwright
zenika/alpine-chrome@sha256:c3a832060aa529bd0ca87bc60ff669015bcc5c8d5138c25b729aee5c2ae8cad6
```
2. Execute :
```
docker container run -it --rm -v $(pwd):/usr/src/app --cap-add=SYS_ADMIN zenika/alpine-chrome:with-playwright npm start
```
3. See error
```
> zenika-alpinechrome-withplaywright@1.0.0 start /usr/src/app
> node src/useragent

internal/modules/cjs/loader.js:969
  throw err;
  ^

Error: Cannot find module 'playwright-chromium'
Require stack:
- /usr/src/app/src/useragent.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:966:15)
    at Function.Module._load (internal/modules/cjs/loader.js:842:27)
    at Module.require (internal/modules/cjs/loader.js:1026:19)
    at require (internal/modules/cjs/helpers.js:72:18)
    at Object.<anonymous> (/usr/src/app/src/useragent.js:1:22)
    at Module._compile (internal/modules/cjs/loader.js:1138:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1158:10)
    at Module.load (internal/modules/cjs/loader.js:986:32)
    at Function.Module._load (internal/modules/cjs/loader.js:879:14)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:71:12) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [ '/usr/src/app/src/useragent.js' ]
}
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! zenika-alpinechrome-withplaywright@1.0.0 start: `node src/useragent`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the zenika-alpinechrome-withplaywright@1.0.0 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm WARN Local package.json exists, but node_modules missing, did you mean to install?

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/chrome/.npm/_logs/2020-06-17T10_09_15_847Z-debug.log

```

**What is the expected behavior?**
As per the documentation: 
> A example-chromium.png file will be created in your with-playwright folder.

**What is the actual behavior?**
> The above output - no png file created.

**Possible solution**
(Not found yet)

**Logs**
Log file [2020-06-17T10_09_15_847Z-debug.log](https://github.com/Zenika/alpine-chrome/files/4791786/2020-06-17T10_09_15_847Z-debug.log)

**Versions**

| Software      | Version(s)   | How-to                                                                                |
| ------------- | ------------ | ------------------------------------------------------------------------------------- |
| Chromium      | Chromium 83.0.4103.61 | `docker run -it --rm --entrypoint "" zenika/alpine-chrome chromium-browser --version` |
| Image         | zenika/alpine-chrome@sha256:c3a832060aa529bd0ca87bc60ff669015bcc5c8d5138c25b729aee5c2ae8cad6       | `docker image inspect --format='{{index .RepoDigests 0}}' zenika/alpine-chrome`       |
| Docker Engine Client | 19.03.11        | `docker version`                                                                      |
| Docker Engine Server | 19.03.5        | `docker version`                                                                      |
| Docker Machine | 0.16.2, build bd45ab1        | `docker-machine version`                                                                      |
| Darwin Kernel Version | 19.3.0 ||
| MacOS version | Catalina 10.15.3 (19D76) ||


**Additional context**
I also tried to replace ` require("playwright-chromium");` with ` require("playwright");` nothing better.
I found the [following issue](https://github.com/microsoft/playwright/issues/2540) but it's not the same version so I'm not sure it's relevant.

--- a/README.md
+++ b/README.md
@@ -162,10 +162,10 @@ Like ["Puppeteer"](https://pptr.dev/#?product=Puppeteer&version=v1.15.0&show=api
 Go to the `with-playwright` folder and launch the following command:
 
 ```
-docker container run -it --rm -v $(pwd):/usr/src/app --cap-add=SYS_ADMIN zenika/alpine-chrome:with-playwright npm start
+docker container run -it --rm -v $(pwd)/src:/usr/src/app/src --cap-add=SYS_ADMIN zenika/alpine-chrome:with-playwright node src/useragent.js
 ```
 
-A `example-chromium.png` file will be created in your `with-playwright` folder.
+A `example-chromium.png` file will be created in your `with-playwright/src` folder.
 
 # How to use with WebGL
 

--- a/with-playwright/src/useragent.js
+++ b/with-playwright/src/useragent.js
@@ -21,6 +21,6 @@ const { chromium } = require("playwright-chromium");
   const context = await browser.newContext();
   const page = await context.newPage();
   await page.goto("http://whatsmyuseragent.org/");
-  await page.screenshot({ path: `example-chromium.png` });
+  await page.screenshot({ path: `src/example-chromium.png` });
   await browser.close();
 })();

